### DISCUSSION TITLE
Full width cards

### DISCUSSION THREAD

**Discussion by ThomasMahon** [GitbookIO/community#61] [Feature Requests]
### What's your feature/idea?

Cards only have two sizes: adding full width would be a welcomed addition.

### What's your desired outcome?

Card which span the full width of the page.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [ ] Medium
- [X] Low

**Comment by taszap** [comment]
Hey @ThomasMahon we have shipped a change which allows you to set cards block to full width too! 
<img width="246" alt="image" src="https://github.com/GitbookIO/community/assets/63643069/bdac16be-31d5-431d-ba8a-3562b15d61ba">
